shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -292,46 +290,24 @@ - (void)dealloc {
                                                       object:self
                                                     userInfo:nil];
 
-  // It will be destroyed and invalidate its weak pointers
-  // before any other members are destroyed.
-  _weakFactory.reset();
-
   /// nil out weak references.
   [_registrars
       enumerateKeysAndObjectsUsingBlock:^(id key, FlutterEngineRegistrar* registrar, BOOL* stop) {
         registrar.flutterEngine = nil;
       }];
 
-  [_labelPrefix release];
-  [_initialRoute release];
-  [_pluginPublications release];
-  [_registrars release];
-  _binaryMessenger.parent = nil;
-  _textureRegistry.parent = nil;
```

Comment:
I _think_ these two are still needed?

Reply:
Oh crap -- yes ".parent". Thanks for spotting.